lib/mayaUsd/ufe/UsdContextOps.cpp

```diff
@@ -54,6 +56,58 @@ class SetVariantSelectionUndoableCommand : public Ufe::UndoableCommand
     const std::string fNewSelection;
 };
 
+//! \brief Undoable command for add new prim
+class AddNewPrimUndoableCommand : public Ufe::UndoableCommand
+{
+public:
+    AddNewPrimUndoableCommand(const MayaUsd::ufe::UsdSceneItem::Ptr& usdSceneItem,
+                              const Ufe::ContextOps::ItemPath& itemPath)
```

Comment:
Looks like you're only using itemPath[1].  If that's the case, couldn't we just pass that in, and give it a more readable name, i.e. const std::string& primType, or something similar?

Reply:
O.K.